### ISSUE TITLE
Would it be possible to bring more features of LT browser extension in, like synonym suggestion?

### ISSUE THREAD

**Issue by lethefrost** [Clemens-E/obsidian-languagetool-plugin#59]
Hi, thank you for this plugin! It's quite a great job.

I am wondering would it be possible to get some more features of LT? For example, when we type in a browser with LT extension installed, and we select a word, there's a pop-up window shows the alternatives/ synonyms of that selected word. See the screenshot attached below: 
<img width="863" alt="image" src="https://user-images.githubusercontent.com/47198693/169521214-f14c4e01-8aef-4f1f-b2da-253bd537eda9.png">

**Comment by froses** [comment]
I agree. It is important for writers to have access to synonyms. Please include this feature.

And thank you so much for this incredibly useful plugin.

**Comment by xplosionmind** [comment]
I totally second this!

**Comment by Clemens-E** [comment]
None of the other features are publicly documented, some are not usable at all without reverse engineering.
The LanguageTool CEO Told me the Synonym API is not "documented" and they can't give me a guarantee that it doesn't change or stop working, but I'm allowed to use it
